geode-core/src/main/java/org/apache/geode/internal/cache/InitialImageOperation.java

```diff
@@ -1589,14 +1589,13 @@ public boolean goWithFullGII(DistributedRegion rgn, RegionVersionVector requeste
         }
         return true;
       }
-      if (!rgn.getVersionVector().isRVVGCDominatedBy(requesterRVV)) {
+      if (!rgn.getVersionVector().isRvvGcDominatedByRequesterRvv(requesterRVV)) {
```

Comment:
Nitpick:  Revert this method name change.  Version vectors don't know anything about "requesters".  Dominance is a mathematical vector operation.